modular_skyrat/code/modules/mob/living/carbon/human/species_types/speciessnowflake.dm

```diff
@@ -0,0 +1,95 @@
+	fluff_desc = "Humans come from the Sol system, and are the commanding species of both NanoTrasen and SolGov."
+
+/datum/species/angel
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Not much is know about angels, but they are most commonly associated with christian mythology."
+
+/datum/species/corporate
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Corporate robots are simply advanced corporate androids employed by NT. Most of the time, they are high ranking CC officers."
+
+/datum/species/dullahan
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Not much is known about the dullahan, but they sure know how to make someone lose their minds."
+
+/datum/species/dwarf
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Dwarves are very similar to humans in biology, but they seem to have completely split from them as a species."
+
+/datum/species/mammal
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Anthropomorphs encompass a lot of the know sapient species by SolGov, and their backgrounds always vary wildly."
+
+/datum/species/jelly
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Slimepeople are the result of scientifical experimentation on slimes, which NanoTrasen has heavily invested in. At the current day, most slimepeople are simply consumers of brands of slime genemods sold by NanoTrasen."
+
+/datum/species/plasmaman
+	screamsounds = list('modular_citadel/sound/voice/scream_skeleton.ogg')
+	fluff_desc = "Plasmamen are the result of catastrophic experimentation with plasma. They are very fragile and in most circumstances require external support to survive, and for this reason are considered most of the time simply cheap workforce."
+
+/datum/species/pod
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Podpeople are simply plantified clones of other sapient species - No one knows where podplants were found for sure, but podpeople have since formed splinter groups about glorifying and preserving nature, coexisting with advanced technology."
+
+/datum/species/shadow
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Shadowpeople are considered ashlanders that were somehow cursed by the necropolis. They are doomed to live in the dark, as light burns their flesh away."
```

Comment:
"Shadowpeople are by and large, mostly unknown. Their origins are a mystery to almost all of civilized spaced - and with mystery, comes Myths. The most popular of these myths is that Shadowpeople were once dwellers of a distant molten world, who lived in harmony with dark forces who controlled the underground of the planet. One day, however it may have happened, these peoples angered the dark Gods of the molten planet, who cursed them to forevermore live in eternal darkness, lest their forms waste away to the ash they had been born from."